index.html

```diff
@@ -73,8 +73,99 @@
       </div>
     </div>
   </header>
-  <main></main>
+
+    <main>
+     <section class="product">
+      <div class="carousel">
+      <div class="product-carousel">
+        <div class="carousel-img">
+          <div class="carousel-item product-banner" style="background-image: url('image/Banner.jpg')"></div>
+          <div class="carousel-item product-banner" style="background-image: url('image/Banner.jpg')"></div>
+          <div class="carousel-item product-banner" style="background-image: url('image/Banner.jpg')"></div>
+        </div>
+      <div class="indicators">
+        <span class="dot" onclick="currentSlide(0)"></span>
+        <span class="dot" onclick="currentSlide(1)"></span>
+        <span class="dot" onclick="currentSlide(2)"></span>
+      </div>
+     </div>
+     <button class="prev" onclick="moveSlide(-1)">&#10094;</button>
+     <button class="next" onclick="moveSlide(1)">&#10095;</button>
```

Comment:
Igual que en un comentario anterior, hazlo con clases `js-`
```suggestion
     <button class="prev js-move-slide" data-direction="-1">&#10094;</button>
     <button class="next js-move-slide" data-direction="1">&#10095;</button>
```